driver/pace/driver/initialization.py

```diff
@@ -136,15 +136,25 @@ class RestartConfig(Initializer):
 
     path: str = "."
     start_time: datetime = datetime(2000, 1, 1)
+    fortran_data: bool = False
```

Comment:
Instead of having this flag, it should be possible to automatically look at the path and determine whether it's a fortran restart or pace restart directory. They look very different.